### ISSUE TITLE
Lambda Indentation not enforced

### ISSUE THREAD

**Issue by laktak** [dotnet/roslyn#43035]
```
  Test(() =>
    {
      return "test";
    });

  Test(() =>
  {
    return "test";
  });
```

When I run this code through dotnet-format version '3.3.111304' it will not be reformatted.

I'd expect it to use either the first or the second style ... but not allow both.

There appears to be no `.editorconfig` setting to enforce this.

**Comment by JoeRobich** [comment]
Similar to https://github.com/dotnet/roslyn/issues/32438

**Comment by laktak** [comment]
I tried to look up the coding style recommended by Microsoft (for the default case) but such a thing does not appear to exist (for multiline lambdas) - is that correct?

**Comment by CyrusNajmabadi** [comment]
I don't believe there is any recommended coding style.  For example, its common to see:

```
Test(() =>
{
    return "test";
});
```

Or

```
Test(() =>
     {
         return "test";
     });
```
